sdk/communication/Azure.Communication.Calling.Server/src/CallClient.cs

```diff
@@ -0,0 +1,491 @@
+                    new PhoneNumberIdentifierModel(alternateCallerId),
+                    operationContext,
+                    null,
+                    cancellationToken).ConfigureAwait(false);
+            }
+            catch (Exception ex)
+            {
+                scope.Failed(ex);
+                throw;
+            }
+        }
+
+        /// <summary> Add a participant to the call. </summary>
+        /// <param name="callLegId"> The call leg id. </param>
+        /// <param name="participant"> The identity of participant to be added to the call. </param>
+        /// <param name="alternateCallerId">The phone number to use when adding a pstn participant.</param>
+        /// <param name="operationContext">The operation context. </param>
+        /// <param name="cancellationToken"> The cancellation token. </param>
+        /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
+        /// <exception cref="ArgumentNullException"> <paramref name="participant"/> is null. </exception>
+        public virtual Response AddParticipant(string callLegId, CommunicationIdentifier participant, string alternateCallerId = default, string operationContext = default, CancellationToken cancellationToken = default)
+        {
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(CallClient)}.{nameof(AddParticipant)}");
+            scope.Start();
+            try
+            {
+                Argument.AssertNotNull(participant, nameof(participant));
+                Argument.AssertNotNullOrEmpty(alternateCallerId, nameof(alternateCallerId));
+
+                return RestClient.InviteParticipants(callLegId,
+                    new List<CommunicationIdentifierModel>() { CommunicationIdentifierSerializer.Serialize(participant) },
+                    new PhoneNumberIdentifierModel(alternateCallerId),
+                    operationContext,
+                    null,
+                    cancellationToken);
+            }
+            catch (Exception ex)
+            {
+                scope.Failed(ex);
+                throw;
+            }
+        }
+
+        /// <summary> Remove a participant from the call. </summary>
+        /// <param name="callLegId"> The call leg id. </param>
+        /// <param name="participantId"> The participant id. </param>
+        /// <param name="cancellationToken"> The cancellation token. </param>
+        /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
+        /// <exception cref="ArgumentNullException"> <paramref name="participantId"/> is null. </exception>
+        public virtual async Task<Response> RemoveParticipantAsync(string callLegId, string participantId, CancellationToken cancellationToken = default)
```

Comment:
Why is this receiving `participantId` as `string` instead of a `CommunicationIdentifier`?

Reply:
On the server side, we use participantId to find the participant that need to be removed from the conversation participants list.

Reply:
![image](https://user-images.githubusercontent.com/19615595/120721230-3e015780-c482-11eb-87b8-16f9433a3cda.png)
When we add participant, it takes in `CommunicationIdentifier` but we take in strong for remove participant. I think we should have some consistency here. We can still take in type `CommunicationIdentifier` and pass its `id` to the service